src/Event/PostReadEvent.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace ApiPlatform\Core\Event;
+
+use ApiPlatform\Core\DataProvider\CollectionDataProviderInterface;
+use ApiPlatform\Core\DataProvider\ItemDataProviderInterface;
+use ApiPlatform\Core\DataProvider\SubresourceDataProviderInterface;
+use ApiPlatform\Core\Identifier\IdentifierConverterInterface;
+use ApiPlatform\Core\Serializer\SerializerContextBuilderInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class PostReadEvent extends Event
+{
+    const NAME = 'api_platform.post_read';
+
+    /** @var CollectionDataProviderInterface */
+    protected $collectionDataProvider;
+    /** @var ItemDataProviderInterface */
+    protected $itemDataProvider;
+    /** @var SubresourceDataProviderInterface */
+    protected $subresourceDataProvider;
+    /** @var SerializerContextBuilderInterface */
+    protected $serializerContextBuilder;
+    /** @var IdentifierConverterInterface */
+    protected $identifierConverter;
+
+    public function __construct(CollectionDataProviderInterface $collectionDataProvider, ItemDataProviderInterface $itemDataProvider, SubresourceDataProviderInterface $subresourceDataProvider = null, SerializerContextBuilderInterface $serializerContextBuilder = null, IdentifierConverterInterface $identifierConverter = null)
```

Comment:
The event must be a value object, it should have no behavior, and should not depend of services. The behaviors (and the deps to the services) must be injected directly in the subscribers/listeners.

To me events, should just contain the current value of the `data` attribute.